src/clients/inst_target_part.rb

```diff
@@ -443,11 +443,12 @@ def main
             Builtins.y2milestone("proposed partitions:%1", @pl)
             if StorageProposal.GetProposalHome &&
                 Builtins.size(Builtins.filter(@pl) do |p|
-                  Ops.get_string(p, "mount", "") == "/home"
+                  Ops.get_string(p, "mount", "") == StorageProposal.GetHomePath()
                 end) == 0
               @ok = false
-              reason = _(
-                "Not enough space available to propose separate /home."
+              reason = Builtins.sformat(
```

Comment:
Why to use `Builtinst.sformat` (with its own placeholder format) instead of the more standard Ruby's `format` (which uses standard strformat placeholders).

Reply:
I think I wanted to be consistent with the rest of the translatable strings:
```console
$ grep -c '%1' storage.pot
295
$ grep -c '%s' storage.pot
1
```